drivers/gpu/drm/bridge/cdns-mhdp.c

```diff
@@ -577,6 +577,17 @@ static u8 eq_training_pattern_supported(struct cdns_mhdp_host host,
 	return fls(host.pattern_supp & sink.pattern_supp);
 }
 
+static bool get_ssc_supported(struct cdns_mhdp_device *mhdp)
+{
+	bool ssc = false;
+
+	/* test if SSC is supported by both sides */
+	if ((mhdp->host.ssc) && (mhdp->sink.ssc))
+		ssc = true;
+
+	return ssc;
+}
```

Comment:
Prefix the function with "mhdp".

Also, this can be a one-liner:
`return (mhdp->host.ssc) && (mhdp->sink.ssc);`